testsupport/init.go

```diff
@@ -229,10 +229,6 @@ func WaitForDeployments(t *testing.T) wait.Awaitilities {
 		err = initHostAwait.WaitUntilBaseNSTemplateTierIsUpdated(t)
 		require.NoError(t, err)
 
-		// check that the default user tier exists and is updated to the current version, an outdated version is applied from deploy/e2e-tests/usertier-base.yaml as
-		// part of the e2e test setup make target for the purpose of verifying the user tier update mechanism on startup of the host operator
-		err = initHostAwait.WaitUntilBaseUserTierIsUpdated(t)
-		require.NoError(t, err)
```

Comment:
why removing this? Correct me if I'm wrong, it doesn't seem to be related to your changes

Reply:
You are right, i just forgot to put it back. done now